### PR TITLE
HMA-6280 November rate changes

### PR DESCRIPTION
**Pull request by chrisob55**

### CODE REVIEW

File: src/jvmTest/kotlin/uk/gov/hmrc/calculator/ParameterizedCalculatorTests.kt [hunk 1]
```diff
@@ -130,11 +129,65 @@ internal class ParameterizedCalculatorTests {
...
+            isPensionAge = inputIsPensionAge
+        ).run()
+
+        assertEquals(expectedCountry, response.country, "Country did not match")
+        assertEquals(expectedIsKCode, response.isKCode)
+
+        val yearlyPeriod = response.yearly
+        println(inputTaxCode)
+        println(yearlyPeriod.prettyPrintDataClass())
+        println("expectedYearlyNiEmployer=$expectedYearlyNiEmployer, yearlyPeriod.employersNI=$yearlyPeriod.employersNI")
```
Comment (ngoulongkam) [inline src/jvmTest/kotlin/uk/gov/hmrc/calculator/ParameterizedCalculatorTests.kt:165]: Do you want to keep these println?

File: src/commonTest/kotlin/uk/gov/hmrc/calculator/CalculatorTests.kt [hunk 2]
```diff
@@ -31,13 +33,14 @@ import kotlin.test.assertTrue
...
     @Test
-    fun `GIVEN 2021 WHEN all supplied data valid THEN calculates response`() {
+    fun `GIVEN TWENTY_TWENTY_ONE WHEN 40000 THEN calculates response`() {
         val result = Calculator(
             taxCode = "1257L",
             wages = 40000.0,
             payPeriod = PayPeriod.YEARLY,
             taxYear = TaxYear.TWENTY_TWENTY_ONE
         ).run()
+        Logger.i(result.prettyPrintDataClass())
```
Comment (ngoulongkam) [inline src/commonTest/kotlin/uk/gov/hmrc/calculator/CalculatorTests.kt:43]: Did you want to keep these Loggers? (and the others in the same class)

**Review by ngoulongkam — CHANGES_REQUESTED** [review]

File: src/jvmTest/kotlin/uk/gov/hmrc/calculator/ParameterizedCalculatorTests.kt (hunk 1 above)
Comment (chrisob55) [reply src/jvmTest/kotlin/uk/gov/hmrc/calculator/ParameterizedCalculatorTests.kt:165]: For both of these, actually yes. It helps with diagnosing failures on Bitrise as KMP doesn't print very nicely with Gradle etc

File: src/commonTest/kotlin/uk/gov/hmrc/calculator/CalculatorTests.kt (hunk 2 above)
Comment (chrisob55) [reply src/commonTest/kotlin/uk/gov/hmrc/calculator/CalculatorTests.kt:43]: As above

**Review by ngoulongkam — APPROVED** [review]

**Pull request merged**